integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -8346,6 +8349,11 @@ msgstr ""
 "Sie haben nicht die nötige Berechtigung, um Veranstaltungen zu bearbeiten "
 "oder zu erstellen."
 
+#: cms/views/events/event_list_view.py cms/views/pages/page_tree_view.py
+#: cms/views/pois/poi_list_view.py
+msgid "German (Plain)"
+msgstr "Deutsch (leicht)"
```

Comment:
Am I understanding this correctly that for the municipality we just give them plain German instead of easy German but under the same language label or should the name of the language be renamed to Deutsch (einfach)? @dkehne